react/react-training/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_BASE_URL=http://localhost:3001/api/
```

Comment:
This file contains sensible info, it should not be commited. Add it to .gitignore file